README.md

```diff
@@ -103,77 +103,95 @@ npm install
 
 #### Infusion All Build
 
-Will include all of Infusion. The source files packaged along with the single concatenated js file will include all of
-the demos and unit tests. This is a good choice if you are trying to learn Infusion.
+Will include all of Infusion, including third-party dependencies. The source files packaged along with the single
+concatenated JavaScript file will include all of the demos and unit tests. This is a good choice if you are trying to
+learn Infusion.
 
 ```bash
-grunt
+npm run build:all
 ```
 
-##### Custom Build
-
-Will only include the modules you request, and all of their dependencies, minus any that are explicitly excluded. Unlike
-the "all" build, none of the demos or tests are included with a custom package.
+By default all packages are minified. Running the build script with the `:source` suffix will allow you to maintain
+readable spacing and comments.
```

Comment:
If we switch to Tenser is this still true? That is, will the comments be preserved or stripped out?

Reply:
Looks like we need to explicitly enable them for the expanded output: https://terser.org/docs/api-reference#output-options

Can you add a comment to that effect to the terser PR, @jobara?